### ISSUE TITLE
Error on TypeScript build: standard modules has no default export

### ISSUE THREAD

**Issue by frankie567** [google/google-auth-library-nodejs#381]
Hi,

I'm currently building a TypeScript project using `googleapis` library (version `32.0.0`, with `google-auth-library` version `1.6.0`). Whenever I try to build my project, here are the errors I get:

```bash
node_modules/google-auth-library/build/src/auth/googleauth.d.ts(18,8): error TS1192: Module '"fs"' has no default export.
node_modules/google-auth-library/build/src/auth/googleauth.d.ts(19,8): error TS1192: Module '"http"' has no default export.
node_modules/google-auth-library/build/src/auth/googleauth.d.ts(20,8): error TS1192: Module '"stream"' has no default export.
node_modules/google-auth-library/build/src/auth/jwtaccess.d.ts(17,8): error TS1192: Module '"stream"' has no default export.
node_modules/google-auth-library/build/src/auth/jwtclient.d.ts(18,8): error TS1192: Module '"stream"' has no default export.
node_modules/google-auth-library/build/src/auth/oauth2client.d.ts(18,8): error TS1192: Module '"http"' has no default export.
node_modules/google-auth-library/build/src/auth/refreshclient.d.ts(17,8): error TS1192: Module '"stream"' has no default export.
```

It seems that TypeScript expects the `import * as http from 'http'` syntax for the standard modules.

I tried with both TypeScript **2.8** and **2.9**, same issue. Node version: **8.11**.

Could you look into this? Is there something wrong with my build configuration?

Best regards,

François Voron

**Comment by nolanmar511** [comment]
I'm seeing the same issue on continuous tests for `@google-cloud/profiler` with node 6, and can reproduce locally with node 8 and 10 when I modify the project's package-lock.json to use v1.6.0. I'm using typescript 2.8

I think there's a problem with v1.6.0

**Comment by JustinBeckwith** [comment]
Taking a first look.  Can you try running your TypeScript compiler with `--esModuleInterop` to see if that fixes the issue?

**Comment by JustinBeckwith** [comment]
cc @ofrobots @google/google-node-team

**Comment by JustinBeckwith** [comment]
`esModuleInterop` was introduced as a default for `gts` [here](https://github.com/google/ts-style/commit/e6ebb5b713a7202c5bafd23ed6b363350749e023#diff-19e170b73889957f2d2c69de5a7a5d7a).  We upgraded to `gts@0.6`, which included the move to synthetic imports in #372.  Still figuring out what's up.

**Comment by frankie567** [comment]
Yes, setting the `esModuleInterop` flag to true solves the issue.
